crates/egui_kittest/src/snapshot.rs

```diff
@@ -49,6 +137,15 @@ impl SnapshotOptions {
         self.output_path = output_path.into();
         self
     }
+
+    /// Change the number of pixels that can differ before the snapshot is considered a failure.
+    /// Preferably, you should use `threshold` to control the sensitivity of the image comparison.
+    /// As a last resort, you can use this to allow a certain number of pixels to differ.
+    #[inline]
+    pub fn diff_threshold(mut self, diff_threshold: impl Into<i32>) -> Self {
```

Comment:
Why `i32`? Setting this to a negative number makes no sense. `usize` makes most sense imho